docs/storage.md

```diff
@@ -0,0 +1,44 @@
+# Timeasr data management
+
+Timeasr data is stored locally in the browser.
+
+## Log format (database)
+
+Timestamp is a minute level epoch (minutes since 1 January 1970.)
+
+| Timestamp    | Type  | Value regex       | Example        | Notes         |
+|--------------|-------|-------------------|----------------|---------------|
+| 201911231515 | TIMER | (start\|stop)     | start          |               |
```

Comment:
Storage should be revised based on different purposes. Time log data should be separated from workdays and settings. Settings might be kept in local storage instead of transaction logs.